ibis/backends/base/sql/__init__.py

```diff
@@ -22,7 +22,6 @@
     import pandas as pd
     import pyarrow as pa
 
-raise RuntimeError("Temporarily make the SQL backends dysfunctional")
```

Comment:
Can we keep it while all the backends get ported?

Reply:
Not really, I need it to be able to use udfs in impala